dist/ExtUtils-ParseXS/lib/ExtUtils/ParseXS.pm

```diff
@@ -839,11 +848,14 @@ EOM
       unshift(@args, $arg0);
     }
 
-    my $extra_args = 0;
-    my @args_num = ();
     my $num_args = 0;
     my $report_args = ''; # the arg's description as used by croak()
     my $ellipsis;
+    my $min_args;
+
+    {
+    my $extra_args = 0;
+    my @args_num = ();
```

Comment:
I believe that everything between lines 856 and 908 (exclusive) should be indented 2(?) spaces -- unless that's handled in a later commit.

Reply:
The re-indentation is done in the follow-up commit.